src/drivers/serac.cpp

```diff
@@ -158,16 +148,42 @@ int main(int argc, char* argv[])
   solid_solver.setDisplacementBCs(ess_bdr, disp_coef);
   solid_solver.setTractionBCs(trac_bdr, traction_coef);
 
+  // neo-Hookean material parameters
+  double mu, K;
+  inlet->get("nonlinear_solid/mu", mu);  // has default value
+  inlet->get("nonlinear_solid/K", K);    // has default value
+
   // Set the material parameters
   solid_solver.setHyperelasticMaterialParameters(mu, K);
 
-  // Set the linear solver parameters
-  if (gmres_solver == true) {
+  // Solver parameters
+  serac::NonlinearSolverParameters nonlin_params;
+  inlet->get("nonlinear_solid/solver/nonlinear/rel_tol", nonlin_params.rel_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/nonlinear/abs_tol", nonlin_params.abs_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/nonlinear/max_iter", nonlin_params.max_iter);        // has default value
+  inlet->get("nonlinear_solid/solver/nonlinear/print_level", nonlin_params.print_level);  // has default value
+
+  serac::LinearSolverParameters lin_params;
+  inlet->get("nonlinear_solid/solver/linear/rel_tol", lin_params.rel_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/linear/abs_tol", lin_params.abs_tol);          // has default value
+  inlet->get("nonlinear_solid/solver/linear/max_iter", lin_params.max_iter);        // has default value
+  inlet->get("nonlinear_solid/solver/linear/print_level", lin_params.print_level);  // has default value
+
+  // solver input args
+  std::string solver_type;
+  inlet->get("nonlinear_solid/solver/linear/solver_type", solver_type);  // has default value
+  std::cout << "~~ Solver Type = " << solver_type << std::endl;
+  if (solver_type == "gmres") {
     lin_params.prec       = serac::Preconditioner::BoomerAMG;
     lin_params.lin_solver = serac::LinearSolver::GMRES;
-  } else {
+  } else if (solver_type == "slu") {
     lin_params.prec       = serac::Preconditioner::Jacobi;
     lin_params.lin_solver = serac::LinearSolver::MINRES;
```

Comment:
This should be `serac::LinearSolver::SuperLU`.

Reply:
Yea that was a remnant of my long standing branch and grabbing the wrong conflict.  Should i just remove superlu for minres or keep both?